timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -228,27 +228,30 @@ types:
       LeaderTimes:
         fields:
           leaderTimes: map<Namespace, LeaderTime>
-      TimestampLeaseRequest:
-         fields:
-           requestId:
-             type: uuid
-             safety: safe
-           numFreshTimestamps:
-             type: integer
-             safety: safe
+      RequestId:
+        alias: uuid
+        safety: safe
       TimestampLeaseRequests:
-         alias: map<TimestampLeaseName, TimestampLeaseRequest>
+          fields:
+            requestsId: RequestId
+            numFreshTimestamps: map<TimestampLeaseName, integer>
+      TimestampLeasesRequest:
+          alias: list<TimestampLeaseRequests>
       MultiClientTimestampLeaseRequest:
-         alias: map<Namespace, TimestampLeaseRequests>
+         alias: map<Namespace, TimestampLeasesRequest>
       TimestampLeaseResponse:
          fields:
            minLeased: Long
-           leaseGuarantee: LeaseGuarantee
            freshTimestamps: ConjureTimestampRange
       TimestampLeaseResponses:
```

Comment:
The request/response names are now slightly confusing.

Reply:
I suggested some alternatives.